grype/presenter/json/presenter_test.go

```diff
@@ -140,17 +122,14 @@ func TestEmptyJsonPresenter(t *testing.T) {
 
 }
 
-func TestPresenter_Present_NewDocumentSorted(t *testing.T) {
-	_, matches, packages, context, metadataProvider, appConfig, dbStatus := internal.GenerateAnalysis(t, internal.ImageSource)
-	doc, err := models.NewDocument(clio.Identification{}, packages, context, matches, nil, metadataProvider, appConfig, dbStatus)
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	if !sort.IsSorted(models.MatchSort(doc.Matches)) {
-		t.Errorf("expected matches to be sorted")
-	}
-}
+// TODO: fix me
+//func TestPresenter_Present_NewDocumentSorted(t *testing.T) {
```

Comment:
Should this get deleted or uncommented (unclear to me what the issue is that needs fixing)?

Reply:
I'll remove this test, I ended up checking that sorting is occurring in the `models` package instead. We really don't need tests in each presenter package for checking sorting anymore.